autorest/codegen/serializers/__init__.py

```diff
@@ -93,14 +98,17 @@ def _serialize_and_write_package_files_proc(**kwargs: Any):
             for template_name in package_files:
                 file = template_name.replace(".jinja2", "")
                 output_name = out_path / file
-                if not self._autorestapi.read_file(output_name) or file == 'setup.py':
+                if not self._autorestapi.read_file(output_name) or file in _REGENERATE_FILES:
```

Comment:
just to confirm: if a user passes any template / file inside `--package-mode=my-custom-folder/`, that will override any of the templates we have in autorest?

Reply:
If user passes folder, autorest will use those files instead of built-in template files. This logic is to permit to override specific files when rerun autorest.